### ISSUE TITLE
[Very easy feature request] Make time representation in image saves ISO compliant

### ISSUE THREAD

**Issue by coughingmouse** [SimpleMobileTools/Simple-Draw#170]
Now, image is saved as "image_YYYY-MM-DD_hh-mm-ss" and so on. 
However, it would be better if it were "image_YYYYMMDDThhmmss" as this is ISO complaint. I.e. "image_20191220T013021.png"

This may look arbitrary to you but using hh-mm-ss for time is just as arbitrary to others as well.

Another possible but not-so-good compliant representation is "image_YYYY-MM-DDThh:mm:ss" but the files saved in this format will not be readable on Windows because of ":".

**Comment by tibbi** [comment]
That suggested version looks too unreadable, I dont really want to add it. You can already give the file a new name before saving. Thanks

**Comment by coughingmouse** [comment]
I would like you to reconsider this issue: the purpose of standardization is not making it readable but rather so that it is unambiguous. 

I, for instance, have never seen someone use hyphens to denote time in my entire life using English and Korean as a native speaker. I could not understand the hh-mm-ss part at first and thought that it was a pointless bunch of randomly generated numerals. Only because I checked the time by coincidence right after saving the image have I noticed what you meant by the numerals.

Also, note that the hh-mm-ss format also resembles the truncated date format YY-MM-DD. Using the truncated date format YY-MM-DD for time is inherently misleading. Because the user is not explicitly informed of the image save format, one has to cleverly guess the meaning behind the numeral. In some cases, the ingrained unambiguity will result in the user identifying the format in a wrong way, and at best confusion resulting from lack of information.

For instance, for users who received hh-mm-ss=19-04-24 on April 5th for a drawing they first started drawing on April 24th and saving on the 25th, one may mistake the format as YY-MM-DD=19-04-24 so that the initial part of the text YYYY-MM-DD=2019-04-24 is the creation date and the latter hh-mm-ss, or YY-MM-DD on the user point of view, is the save date.
